activejob/lib/active_job/base.rb

```diff
@@ -71,8 +70,7 @@ class Base
     include Exceptions
     include Instrumentation
     include Logging
-    include Timezones
-    include Translation
+    include TimezonesAndTranslation
```

Comment:
Not the best name. I wanted to use `CurrentAttributes`, but it can be confusing why it sets only a locale and a timezone.

Reply:
Maybe something like `ExecutionState`?